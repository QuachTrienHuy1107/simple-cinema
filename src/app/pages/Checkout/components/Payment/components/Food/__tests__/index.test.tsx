import * as React from "react";
import { render } from "@testing-library/react";

import { Food } from "..";

jest.mock("react-i18next", () => ({
    useTranslation: () => {
        return {
            t: str => str,
            i18n: {
                changeLanguage: () => new Promise(() => {}),
            },
        };
    },
}));

describe("<Food  />", () => {
    it("should match snapshot", () => {
        const loadingIndicator = render(<Food />);
        expect(loadingIndicator.container.firstChild).toMatchSnapshot();
    });
});
