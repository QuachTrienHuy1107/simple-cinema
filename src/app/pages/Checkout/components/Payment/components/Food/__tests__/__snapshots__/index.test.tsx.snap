// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Food  /> should match snapshot 1`] = `
.c0 {
  width: 100%;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding-right: 20px;
  height: 30px;
}

.c1 h1 {
  text-align: center;
  color: green;
  width: 100%;
  font-size: 2.3rem;
}

.c1 span {
  color: #016f33;
  font-size: 1rem;
  font-weight: 600;
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    <h3>
      <a
        style="margin-bottom: 0px; display: inline-block; color: red;"
      >
        Ch·ªçn combo
      </a>
    </h3>
    <span>
      0
    </span>
  </div>
</div>
`;
